compiler/optimizer/Dominators.hpp

```diff
@@ -53,11 +53,14 @@ class TR_Dominators
    TR_ALLOC(TR_Memory::Dominators)
 
    TR_Dominators(TR::Compilation *, bool post = false);
+   TR_Dominators(TR::Compilation *, TR::CFG* cfg, bool acceptUnreachableBlocks = true, bool post = false);
    TR::Block       *getDominator(TR::Block *);
    int             dominates(TR::Block *block, TR::Block *other);
 
    TR::Compilation * comp()         { return _compilation; }
    bool trace() { return _trace; }
+   bool isValid() { return _isValid; }
```

Comment:
Could you please add a comment to `isValid()` that explains why it's now possible for this to be invalid? e.g. something like this:
```
   // HACK: Dominators should always be valid, but for some reason the benefit
   // inliner can pass a CFG with unreachable blocks.
```